### DISCUSSION TITLE
Import A MVC Project from a different solution

### DISCUSSION THREAD

**Discussion by StevenLiuToExplore** [dotnet/aspnetcore#27510] [Q&A]
Import a mvc project from the same solution,it works!But in some situation,I need to import a mvc project from a different solution,but it doesn't work.It comes "System.Private.ServiceModel" not found.

Are there any option I can import a mvc project from a different solution?

**Comment by engineering87** [comment]
If you are referencing the project directly, ensure that all dependencies (NuGet packages) used in the MVC project are restored and available in the consuming project.
Open the .csproj file of the MVC project in the other solution and check the dependencies (<PackageReference> elements). 
Make sure these dependencies are available in the referencing project’s environment.
